.ci/ci-fast-return.sh

```diff
@@ -252,35 +321,62 @@ check_force_label() {
 		return 0
 	fi
 
-	local_info "Checking labels for PR ${ghprbGhRepository}/${ghprbPullId}"
+	repo="$ghprbGhRepository"
+	pr="$ghprbPullId"
 
-	# Pull the label list for the PR
-	# Ideally we'd use a github auth token here so we don't get rate limited, but to do that we would
-	# have to expose the token into the CI scripts, which is then potentially a security hole.
-	local json=$(curl -sL https://api.github.com/repos/${ghprbGhRepository}/issues/${ghprbPullId}/labels)
+	local result=$(is_label_set "$repo" "$pr" "$label")
+	if [ "$result" -eq 1 ]; then
+		local_info "label '$label' found"
+		echo "1"
+		return 0
+	fi
 
-	install_jq
+	local_info "label '$label' not found"
```

Comment:
@wainersm - is this the line you meant? This is correct because `is_label_set()` "returns" `1` if the label is found. I've quoted returns because all the functions actually return zero but echo a string to stdout as the actual return value. That was the original design to work within the confines of `set -e` so I've retained that to keep the diff minimal.

Reply:
hi @jodh-intel - I made a comment about that line but later and quickly I noticed I said something wrong. I deleted the comment, so never mind. You change looks good to me.